src/components/identity/globalFooter.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import SbEditable from 'storyblok-react';
+import { GlobalFooter as DrGlobalFooter } from "decanter-react";
+
+const GlobalFooter = (props) => (
+  <SbEditable content={props.blok}>
+    <DrGlobalFooter color={props.blok.color}/>
```

Comment:
Does this work? Or do we get propType issues?
```suggestion
    <DrGlobalFooter {...props.blok} />
```

Reply:
It works!